src/server.cpp

```diff
@@ -2685,20 +2685,24 @@ void Server::DeleteClient(session_t peer_id, ClientDeletionReason reason)
 
 void Server::UpdateCrafting(RemotePlayer *player)
 {
+	InventoryList *clist = player->inventory.getList("craft");
+	if (!clist || clist->getSize() < 1)
```

Comment:
getSize can be lesser than zero ? if not just zero check

Reply:
Unlike the `size()` getter functions in container classes, this here returns a `u32`. Changed it to `== 0` now, whereas it will have the same effect.